### DISCUSSION TITLE
Clarify / add guidance to switch / toggle guidance?

### DISCUSSION THREAD

**Discussion by mma1504** [twilio-labs/paste#3068] [Ideas] [answered]
Hello! I'd like to propose updating guidance in Paste for switches / toggles. Most general guidance, including [NNG](https://www.nngroup.com/articles/toggle-switch-guidelines/) suggest that toggles should only be used when the result is **immediate** - no saving required. I've seen the switch component used in situations where this isn't the case, so I think it might be worthwhile documenting this point? LMK if this is something we could discuss!

**Comment by serifluous** [comment]
Hey @mma1504 , you're totally right! That is the correct guidance, and while we've even guided people toward the right usage in Discussions (see https://github.com/twilio-labs/paste/discussions/3054#discussioncomment-5154163), it's been a total oversight that it's left out of our actual documentation page for Switch.

@nkrantz is going to put up a PR to update the docs, and we'll send you a link to a preview of it so you can review/approve. Let us know if you actually want to be the one to write the guidance, though.

**Comment by nkrantz** [comment, marked as answer]
Hey @mma1504 - thanks for pointing this out! [Here's the deploy preview of the updated Switch docs](https://deploy-preview-3062--paste-docs.netlify.app/components/switch#about-switch), with this guidance included in the "About Switch" section. Feel free to leave feedback in [the PR](https://github.com/twilio-labs/paste/pull/3062) directly or here in this thread.

**Reply by mma1504** [reply]
this looks great - thank you so much!
